identifier/identifier_test.go

```diff
@@ -262,3 +263,82 @@ func TestNormalize(t *testing.T) {
 		})
 	}
 }
+
+func TestToValues(t *testing.T) {
```

Comment:
It would be good to have a test case which ensures ordering is preserved. We care about the order of our SANs but that ordering is only being enforced by `.FromCSR()`, so we need to test that the order is preserved by `.ToValues()`.